runtime/lua/vim/lsp.lua

```diff
@@ -1504,9 +1490,7 @@ end
 --- - findstart=0: column where the completion starts, or -2 or -3
 --- - findstart=1: list of matches (actually just calls |complete()|)
 function lsp.omnifunc(findstart, base)
-  if log.debug() then
-    log.debug('omnifunc.findstart', { findstart = findstart, base = base })
-  end
+  log.debug('omnifunc.findstart', { findstart = findstart, base = base })
```

Comment:
Just fyi as I'm not sure if it has an impact in practice: The idea behind the guards is to avoid the allocations - e.g. in this case creating the table.

Reply:
Yeah I'm fully aware and mindful of this. There's one case I've left this guard in (that has a `vim.inspect(CTX)`), but otherwise it's a micro-op which doesn't make sense since performance will likely be bound by RPC IO.